### PR TITLE
fixed config.yml and added readme instructions

### PR DESCRIPTION
**Pull request by jenningshanna**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,21 +2,22 @@ ABQs Financial Wellness Resource Wizard
...
+#Deploying this application:
 
-**Q: What is your level of financial stress?**
+In order to deploy this code you will need to install the Jekyll gem through your command line with `gem install jekyll` (you need Ruby installed to do this).
 
-**A:** Not Stressed
+Once you've done that you should simply be able to run the Jekyll command `jekyll serve` or `jekyll serve --watch` and see the site hosted on your localhost:4000 (with `--watch` it will listen for updates).
 
-**Q: What do you worry about more, paying bills regularly or saving money for a rainy day or big purchase?**
+In order to deploy the site to your own github repo you'll first need to set up your `gh-pages` branch and make that your default branch in your GitHub repositories settings. That will make your site appear at http:[username].github.io/[repo name].
```
Comment (boonrs) [inline README.md]: This is unnecessary. We're only interested in getting them up and running with our codebase.

File: README.md [hunk 2]
```diff
@@ -2,21 +2,22 @@ ABQs Financial Wellness Resource Wizard
...
 
-**A:** Paying Bills
+Lastly, you'll need to set the config.yml file to reflect your new base and sub URLs before you push to `gh-pages`. Example:`url: "http://codeforamerica.github.io/"` and `baseurl: "abq-financial-wellness-wizard"`. Don't forget to set the title and description as well.
```
Comment (boonrs) [inline README.md]: Ditto on this.

**Comment by jenningshanna** [comment]
Ok, removed and fixed conflicts

**Comment by boonrs** [comment]
This won't run for me. To troubleshoot, check out a clean version of the code and verify you can get it to work. A "clean version" means cloning the repo to a new folder, switching to that folder, checking out the imanol branch and following the instructions.

 I get the following error when following the readme directions:

```
D, [2015-09-03T17:14:29.976249 #50433] DEBUG -- : Celluloid 0.17.0 is running in BACKPORTED mode. [ http://git.io/vJf3J ]
jekyll 2.5.3 | Error:  wrong number of arguments (2 for 1)
```

**Comment by jenningshanna** [comment]
I am not getting this error but it does look like the `baseurl` in the `_config.yml` file is causing an issue in the local environment. Not sure what to do about that other than have the git ignore the config file and build it's own on the way down? Also not sure how that would work...

As for the error you got here's what came up in a google search: http://stackoverflow.com/questions/651038/how-do-you-clone-a-git-repository-into-a-specific-folder

**Comment by boonrs** [comment]
@jenningshanna To clarify, you were able to checkout a clean version and run it?

**Comment by jenningshanna** [comment]
@boonrs yes, I was. That's when I had the `baseurl` issue.

**Comment by jenningshanna** [comment]
Something has gone terribly wrong here where all of my Jekyll layout code was deleted. Though it's installed now all the work I did with Imanol to set it up is gone. I don't know if it happened in the revert or what but I cannot find the edits anywhere— Not in a commit or a PR! So I am going to close this and have to redo it next week. Let me know if you know where that node may have fallen off.

**Pull request closed without merging**